src/commands/org/open.ts

```diff
@@ -128,47 +124,20 @@ export class OrgOpenCommand extends SfCommand<OrgOpenOutput> {
       throw err;
     }
 
-    await utils.openUrl(url, {
+    // create a local html file that contains the POST stuff.
+    // for review...there's always an access token, right?
+    const tempFilePath = path.join(tmpdir(), `org-open-${new Date().valueOf()}.html`);
+    await writeFile(tempFilePath, getFileContents(conn.accessToken as string, conn.instanceUrl, retUrl));
+    await utils.openUrl(`file:///${tempFilePath}`, {
       ...(flags.browser ? { app: { name: apps[flags.browser] } } : {}),
-      // only applies on macOS.  open library ignores this property as it is the default behavior on other OS
-      newInstance: true,
+      ...(flags.browser === 'browserPrivate' ? { newInstance: true } : {}),
```

Comment:
`newInstance` is a macos-only feature:
https://github.com/sindresorhus/open?tab=readme-ov-file#newinstance-macos-only

for private you need to use `apps.browserPrivate`:
https://github.com/sindresorhus/open?tab=readme-ov-file#usage

Reply:
yes, but
1. it has no effect when used on other platforms (per `open`'s code)
2. we only need it when doing private (otherwise private doesn't work)

Are you wanting the criteria to be tighter, like... if macos AND private, then `newInstance:true` ?

Reply:
ahh right, nvm then, I see you commented this in your previous incognito PR.